test/integration/integration_test.cc

```diff
@@ -221,6 +221,10 @@ TEST_P(IntegrationTest, RouterUpstreamDisconnectBeforeResponseComplete) {
   testRouterUpstreamDisconnectBeforeResponseComplete();
 }
 
+TEST_P(IntegrationTest, ResponseFramedByConnectionCloseWithReadLimits) {
+  testResponseFramedByConnectionCloseWithReadLimits();
```

Comment:
If this is only used in one place please just put the code inline - it's not totally obvious but I'm trying to push for inline code where things are not reused.

Reply:
Done.